### ISSUE TITLE
Вывод информации об подтверждённости аккаунта

### ISSUE THREAD

**Issue by bezgodov** [bezgodov/olympic#10]
Сделать иконку или вписать текст в структуру [разумно и эстетично] о: подтверждён ли аккаунт или нет.
![screenshot_68](https://cloud.githubusercontent.com/assets/9131239/23842350/6d1aa178-07ff-11e7-8a78-371177d3c03f.png)

**Comment by bezgodov** [comment]
1) При изменение масштаба кнопка наезжает на личную информацию;
2) При просмотре с мобильного устройства, кнопка растягивается на весь экран необходимо сделать по центру, как остальная информация;
![screenshot_98](https://cloud.githubusercontent.com/assets/9131239/23928664/ccb4dbda-096c-11e7-94ee-a07ebf6593b9.png)
![screenshot_99](https://cloud.githubusercontent.com/assets/9131239/23928665/ccb5ba50-096c-11e7-97bf-09e4e88ffa26.png)

**Comment by danilenkodanila** [comment]
@klasterstorm сказал, что это приемлемо

**Comment by bezgodov** [comment]
@danilenkodanila не согласен с вами, так как на планшетах с разрешением [610+-], отображается некорректно. Бывает и такое [см. скриншот].
![screenshot_109](https://cloud.githubusercontent.com/assets/9131239/23997492/422022f4-0a9d-11e7-934b-73681d9c2783.png)
